hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/StoreFileInfo.java

```diff
@@ -168,9 +169,8 @@ private StoreFileInfo(final Configuration conf, final FileSystem fs, final FileS
         this.createdTimestamp = fileStatus.getModificationTime();
         this.size = fileStatus.getLen();
       } else {
-        FileStatus fStatus = fs.getFileStatus(initialPath);
-        this.createdTimestamp = fStatus.getModificationTime();
-        this.size = fStatus.getLen();
+        this.createdTimestamp = this.getFileStatus().getModificationTime();
```

Comment:
Generally, we do not put the 'this.' in front of local method invocation.

Reply:
I will remove it .